README.md

```diff
@@ -65,6 +65,21 @@ which does the equivalent of:
 
 with minor changes to the input/output format used between the commands.
 
+### openQA hook scripts - Call auto-review or investigation steps in openQA after every job is done
+
+openQA supports custom job done hook scripts that can be called whenever a job
+is done, see
+http://open.qa/docs/#_enable_custom_hook_scripts_on_job_done_based_on_result
```

Comment:
```suggestion
https://open.qa/docs/#_enable_custom_hook_scripts_on_job_done_based_on_result
```

Reply:
I see no problem with http for the static documentation webpage. Especially when one is redirected by the browser automatically in most cases anyway

Reply:
This was more for consistency. But I also don't care much. If you like you can merge it as-is.

Reply:
As we have not mentioned "open.qa" anywhere else in this repo I will keep it